This patch provides a means of forcing bridge-utils to compile without sysfs.
Patch by Robin H. Johnson <robbat2@gentoo.org>
--- bridge-utils-1.0.6.orig/configure.in	2005-03-16 12:15:46.000000000 -0800
+++ bridge-utils-1.0.6/configure.in	2005-08-05 12:19:48.000000000 -0700
@@ -26,12 +26,17 @@
 AC_CHECK_FUNCS(if_nametoindex if_indextoname)
 
 dnl Check for libsysfs
+
+AC_ARG_WITH( sysfs,       [  --without-sysfs              Disallow use of libsysfs],[],[])
+
+if test "${with_sysfs}" != "no" ; then
 AC_CHECK_HEADER(sysfs/libsysfs.h,
-	[AC_CHECK_LIB(sysfs, sysfs_open_directory, 
-	              [AC_DEFINE(HAVE_LIBSYSFS)
-		       LIBS="$LIBS -lsysfs"],
-		      [AC_MSG_WARN([Missing sysfs library!])])],
-	[AC_MSG_WARN([Missing /usr/include/sysfs/libsysfs.h])])
+		[AC_CHECK_LIB(sysfs, sysfs_open_directory, 
+			[AC_DEFINE(HAVE_LIBSYSFS)
+			LIBS="$LIBS -lsysfs"],
+			[AC_MSG_WARN([Missing sysfs library!])])],
+		[AC_MSG_WARN([Missing /usr/include/sysfs/libsysfs.h])])
+fi
 
 
 AC_SUBST(KERNEL_HEADERS)
